Library/SharedFunctions.swift

```diff
@@ -367,7 +367,7 @@ public func rewardsCarouselCanNavigateToReward(_ reward: Reward, in project: Pro
 
   let isBacking = userIsBacking(reward: reward, inProject: project)
   let isAvailableForNewBacker = rewardIsAvailable(reward) && !isBacking
-  let isAvailableForExistingBackerToEdit = (isBacking && (reward.hasAddOns || featureNoShippingAtCheckout()))
+  let isAvailableForExistingBackerToEdit = isBacking
```

Comment:
- In the new “no shipping” flow, that is now the standard, backers should be able to edit rewards regardless of if that reward hasAddOns. This is because backers should have access to change the bonus or pledge amount.